src/test/java/br/com/muralis/core/dto/colaborador/geral/FiltroColaboradorDTOTest.java

```diff
@@ -0,0 +1,24 @@
+package br.com.muralis.core.dto.colaborador.geral;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class FiltroColaboradorDTOTest {
+
+	@ParameterizedTest
+	@DisplayName("Valida se data inicio é anterior a data fim")
```

Comment:
Entendo que está se referindo ao atributo, porém, por estar em um displayName, não acha interessante colocar 'Início'

Reply:
Está se referindo a acentuação, correto? Claro, posso corrigir sim! Nesses cenários mais "simples", proponho que use o próprio github para propor a melhoria:

```suggestion
	@DisplayName("Valida se data início é anterior a data fim")
```